contracts/SortitionPoolFactory.sol

```diff
@@ -6,21 +6,25 @@ import "./api/IStaking.sol";
 /// @title Sortition Pool Factory
 /// @notice Factory for the creation of new sortition pools.
 contract SortitionPoolFactory {
+  event SortitionPoolCreated(address sortitionPoolAddress);
+
   /// @notice Creates a new sortition pool instance.
   /// @return Address of the new sortition pool contract instance.
   function createSortitionPool(
     IStaking stakingContract,
     uint256 minimumStake,
     uint256 poolWeightDivisor
   ) public returns (address) {
-    return
-      address(
-        new SortitionPool(
-          stakingContract,
-          minimumStake,
-          poolWeightDivisor,
-          msg.sender
-        )
-      );
+    address sortitionPool = address(
+      new SortitionPool(
+        stakingContract,
+        minimumStake,
+        poolWeightDivisor,
+        msg.sender
+      )
+    );
+
+    emit SortitionPoolCreated(sortitionPool);
```

Comment:
Had to emit event to get the address of the new `sortition pools` in tests.

Reply:
This is fine. Could also help in any web3-related code.